### PR TITLE
Topbar and footer

### PR DESCRIPTION
**Pull request by Adouleur**

### CODE REVIEW

**Comment by MiradorOne** [comment]
image
missing bottom border

File: app/components/layout/HamburgerMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import Currency from "~/components/layout/Currency";
+import Phone from "~/components/layout/Phone";
+import Email from "~/components/layout/Email";
+import Socials from "~/components/layout/Socials";
+
+const HamburgerMenu = () => {
+  const context = useContext(MenuContext);
+  const toggleHamburger = () => {
+    context?.setHamburgerOpen(!context?.hamburgerOpen);
+    console.log(context?.hamburgerOpen);
```
Comment (MiradorOne) [inline app/components/layout/HamburgerMenu.tsx]: remove console.log

**Comment by MiradorOne** [comment]
image
no padding

File: app/components/layout/Navigation.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+interface Props {
+  ulStyle?: string;
+  liStyle?: string;
+}
+
+const Navigation = ({ ulStyle, liStyle }: Props) => {
+  return (
+    <ul className={ulStyle}>
+      <li className={liStyle}>All Cars</li>
```
Comment (MiradorOne) [inline app/components/layout/Navigation.tsx]: Make all menu items as links (could be` to={'/'}`)

File: app/components/layout/HamburgerMenu.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+            <Logo className="ml-4" />
+          </Link>
+        </div>
+        <Navigation ulStyle="mt-[45px] text-gray-900" liStyle="mt-[36px]" />
+      </div>
+      <div className="text-gray-700">
+        <Link to={"/login"} className="flex items-center">
+          <SignInHeader className="mr-1.5" /> Sign In{" "}
+        </Link>
+        <ShipTo isBurger={true} className="mt-6" />
```
Comment (MiradorOne) [inline app/components/layout/HamburgerMenu.tsx:47]: All topheader icons should have` cursor: pointer` even it is not clickable right now. Or make them as` <a>` tag

File: app/components/layout/Socials.tsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  YoutubeHeader,
+} from "~/components/IconComponents";
+interface Props {
+  intent: "24px" | "mr-[20px]" | "mr-[16px]";
+  className?: string;
+  fill?: string;
+  fillOpacity?: string;
+}
+const Socials = ({ intent, className, fill, fillOpacity }: Props) => {
+  console.log(intent);
```
Comment (MiradorOne) [inline app/components/layout/Socials.tsx]: remove

File: app/components/footer.tsx [hunk 5]
```diff
@@ -1,20 +1,65 @@
...
+            fillOpacity="1"
+            className="mt-6"
+          />
+        </div>
+        <div className="grid  min-[320px]:mt-6 min-[320px]:grid-cols-1 md:mt-10	md:grid-cols-3 lg:mt-0 lg:gap-32">
+          <div>
+            <h3 className="font-semibold text-black min-[320px]:mb-4 lg:mb-6">
+              Company
+            </h3>
+            <p className="mb-3">About Us</p>
```
Comment (MiradorOne) [inline app/components/footer.tsx]: Should be as link. Same for bellow

File: app/components/footer.tsx [hunk 6]
```diff
@@ -1,20 +1,65 @@
...
+            <p className="mb-3">Blog</p>
+            <p className="mb-3">FAQ</p>
+          </div>
+          <div>
+            <h3 className="font-semibold text-black min-[320px]:mb-4 lg:mb-6">
+              Cars
+            </h3>
+            <p className="mb-3">Special offers</p>
+            <p className="mb-3">New cars</p>
```
Comment (MiradorOne) [inline app/components/footer.tsx]: you have `New cars` twice

File: app/components/footer.tsx [hunk 7]
```diff
@@ -1,20 +1,65 @@
...
+            <p className="mb-3">New cars</p>
+            <p className="mb-3">Brands</p>
+          </div>
+          <div>
+            <h3 className="font-semibold text-black min-[320px]:mb-4 lg:mb-6">
+              Contacts
+            </h3>
+            <Phone fill="#767676" fillOpacity="1" className="" />
```
Comment (MiradorOne) [inline app/components/footer.tsx]: className is empty

**Review by MiradorOne — CHANGES_REQUESTED** [review]
fix code & ui

File: app/components/layout/Email.tsx [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+}
+const Email = ({ className, fill, fillOpacity }: Props) => {
+  return (
+    <div className={` flex items-center ${className}`}>
+      <SvgEmailHeader
+        className="mr-1.5"
+        fill={fill}
+        fillOpacity={fillOpacity}
+      />
+      <p>request@example.com</p>
```
Comment (MiradorOne) [inline app/components/layout/Email.tsx]: `<a href="mailto:request@example.com">...`

File: app/components/layout/Phone.tsx [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+}
+const Phone = ({ className, fill, fillOpacity }: Props) => {
+  return (
+    <div className={` flex items-center ${className}`}>
+      <SvgWhatsAppHeader
+        className="mr-1.5"
+        fill={fill}
+        fillOpacity={fillOpacity}
+      />
+      <p>+971 55 811 9024</p>
```
Comment (MiradorOne) [inline app/components/layout/Phone.tsx]: `<a href=“tel:555-666-7777”`

**Comment by Adouleur** [comment]
Done! Please check

**Pull request merged**